### DISCUSSION TITLE
Question about plan securities and exercise transactions

### DISCUSSION THREAD

**Discussion by roehst** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#279] [Q&A]
Hi folks,

I have a question about plan securities and how they behave upon exercises.

https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/616d972f4ce99f33ba2729203e56ea5c0acb2017/samples/Transactions.ocf.json#L283-L295

The exercise takes a plan security id as input.

Conceptually, should we decrease the number of shares in the plan security by the amount exercised, or should we create a new security like other transactions have balance securities?

Thanks in advance!

**Comment by pjohnmeyer** [comment]
> Conceptually, should we decrease the number of shares in the plan security by the amount exercised, or should we create a new security like other transactions have balance securities?

If by "decrease the number of shares" you mean "modify the issuance event", no. It also does not create a new balance security because we are still operating on the same certificate. Balance securities are created on events like partial cancellations and partial transfers, in which case the original security is terminated.

To walk through a simple example, let's say a stakeholder is granted 10,000 options, and accepts the grant. They then exercise 100 of those options. The partial transaction log would look like:

```json
{
  "object_type": "TX_PLAN_SECURITY_ISSUANCE",
  "security_id": "abcd123",
  "quantity": "10000"
},
{
  "object_type": "TX_PLAN_SECURITY_ACCEPTANCE",
  "security_id": "abcd123"
},
{
  "object_type": "TX_PLAN_SECURITY_EXERCISE",
  "security_id": "abcd123",
  "quantity": "100",
  "resulting_security_ids": [
    "efgh456"
  ]
},
{
  "object_type": "TX_STOCK_ISSUANCE",
  "security_id": "efgh456",
  "quantity": "100"
}
```

The number of options _available to exercise_ after this series of events is reduced to 9,900 logically, but the number granted is still 10,000 -- the Plan Security Issuance remains unchanged. The only new security created, then, is represented by the stock issuance.

Does this help?
